src/es/components/pages/Register.html

```diff
@@ -527,6 +527,70 @@ <h2>Les Coordonnées du responsable</h2>
                                 name="Data.AccountManagerLastName" type="text" value="" required>
                         </div>
                     </div>
+                    <div class="form-group">
+                        <h5>Pattern validation by clicking submit button</h5>
+                        <label class="control-label" for="Data_AccountManagerLastName">Registration Number *</label>
+                        <div>
+                            <input
+                            class="form-control" 
+                            id="Data_AccountManagerLastName"
+                            name="Data.AccountManagerLastName1" 
+                            type="text" 
+                            pattern="CHE-[0-9]{3}\.[0-9]{3}\.[0-9]{3}"
+                            title="Please add the format like CHE-333.333.333"
+                            maxlength="15" 
+                            placeholder="CHE-333.333.333" 
+                            required>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <h5>Pattern validation inmediatly after user input change</h5>
+                        <label class="control-label" for="Data_AccountManagerLastName">Registration Number *</label>
+                        <div>
+                            <input
+                            class="form-control" 
+                            id="Data_AccountManagerLastName"
+                            name="Data.AccountManagerLastName2" 
+                            type="text" 
+                            value="" 
+                            maxlength="15" 
+                            custom-pattern-validation 
+                            custom-validity-message="Bitte geben Sie die richtige Format CHE-123.123.123" 
+                            placeholder="CHE-XXX.XXX.XXX" 
+                            required>
```

Comment:
It is not possible to enter a hypen or minus (-) to confirm the input.
👇🏻 
![Screenshot 2023-10-05 at 14 48 41](https://github.com/mits-gossau/web-components-toolbox-migrospro/assets/1226135/b89c6c43-c956-42c7-8f20-da35a5c74999)

Reply:
This was a good catch and helped me to recognize an another problem with prevent defaulting some keydown events.
The problem I have recognized is the following:
On different keyboards and on different keyboard language settings, the numbers are on different keyCodes, therefore preventing keydown events based on keyCodes is not  a good idea at all.
I have removed the whole keydown listener, it means that the user is able to write now everything but after the user change the input, the validation triggers.
@ASchmidt1024 let me know what you think, thanks.